packages/testing/src/web/MockProviders.tsx

```diff
@@ -15,9 +15,14 @@ import { MockParamsProvider } from './MockParamsProvider'
 // Import the user's Router from `./web/src/Router.{tsx,jsx}`,
 // we pass the `children` from the user's Router to `./MockRouter.Router`
 // so that we can populate the `routes object` in Storybook and tests.
-const {
-  default: UserRouterWithRoutes,
-} = require('~__REDWOOD__USER_ROUTES_FOR_MOCK')
+let UserRouterWithRoutes: React.FC
```

Comment:
Same naming questions here as above

Reply:
oh i think we don't need to make changes to this file as it's got its own version in the framework package — will undo this change and test it out

Reply:
@arimendelow Sounds good! Let me know how the tests go!

Reply:
ah — i guess because the testing package is still a dependency of the sb framework package, it'll fail on this error:
<img width="979" alt="Screenshot 2024-05-30 at 16 40 46" src="https://github.com/redwoodjs/redwood/assets/16390116/d793e399-87f5-4959-b6c9-93690006d118">

Reply:
(it's a dependency because the sb framework package still uses the stuff shared with Jest, like mockAuth, the graphQL mocks, and MSW — the SB specific stuff lives in the framework package so that the testing package can be cleaned up when we remove webpack)

Reply:
put it back, following the same naming feedback you had on the other file. also added a comment explaining why we need to do the import this way